runtime/integration-tests/src/generic/envs/runtime_env.rs

```diff
@@ -171,6 +191,31 @@ impl<T: Runtime> RuntimeEnv<T> {
 		let mut ext = sp_io::TestExternalities::new(storage);
 
 		ext.execute_with(|| {
+			let accounts = match T::KIND {
+				RuntimeKind::Centrifuge => {
+					runtime_common::evm::precompile::utils::precompile_account_genesis::<
+						centrifuge_runtime::CentrifugePrecompiles,
+					>()
```

Comment:
Not sure if asked before, the following doesn't work?

```suggestion
					runtime_common::evm::precompile::utils::precompile_account_genesis::<
						T::PrecompilesType,
					>()
```

for all runtimes?

Reply:
Will try.

Reply:
```
197 |                         T::PrecompilesType,
    |                         ^^^^^^^^^^^^^^^^^^ the trait `H160Addresses` is not implemented for `<T as pallet_evm::Config>::PrecompilesType`
```

We had this already fix IIRC. But I am not really motived atm to put work into this.